src/compute_bond_local.cpp

```diff
@@ -390,6 +391,11 @@ int ComputeBondLocal::compute_bonds(int flag)
           engrot *= mvv2e;
         }
 
+        // populate map to access per-bond values from atom tags
```

Comment:
I don't like this change (including the change to compute.h). It provides no functionality for compute bond/local and is not controlled by some flag (and thus always consumes resources whether it is used or not) and whoever needs to use it needs to set up compute bond/local in a suitable fashion so that the data is computed. That is rather indirect and difficult to understand from the source code. Why not make computation of the map a self-contained function in fix bond/react?

Reply:
good points. yes, that should work

Reply:
Thanks for making the change.